app/components/channel_actions/manage_members_label/manage_members_label.tsx

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+import {defineMessages, useIntl} from 'react-intl';
+import {Alert, DeviceEventEmitter} from 'react-native';
+
+import {fetchChannelStats, removeMemberFromChannel, updateChannelMemberSchemeRoles} from '@actions/remote/channel';
+import OptionItem from '@components/option_item';
+import {Events, Members} from '@constants';
+import {ManageOptionsTypes} from '@constants/members';
+import {useServerUrl} from '@context/server';
+import {t} from '@i18n';
+import {dismissBottomSheet} from '@screens/navigation';
+import {alertErrorWithFallback} from '@utils/draft';
+
+const {MAKE_CHANNEL_ADMIN, MAKE_CHANNEL_MEMBER, REMOVE_USER} = Members.ManageOptions;
+
+const messages = defineMessages({
+    role_change_error: {
+        id: t('mobile.manage_members.change_role.error'),
+        defaultMessage: 'An error occurred while trying to update the role. Please check your connection and try again.',
+    },
+    make_channel_admin: {
+        id: t('mobile.manage_members.make_channel_admin'),
+        defaultMessage: 'Make Channel Admin',
+    },
+    make_channel_member: {
+        id: t('mobile.manage_members.make_channel_member'),
+        defaultMessage: 'Make Channel Member',
+    },
+    remove_title: {
+        id: t('mobile.manage_members.remove_member'),
+        defaultMessage: 'Remove From Channel',
+    },
+    remove_message: {
+        id: t('mobile.manage_members.message'),
+        defaultMessage: 'Are you sure you want to remove the selected member from the channel?',
+    },
+    remove_cancel: {
+        id: t('mobile.manage_members.cancel'),
+        defaultMessage: 'Cancel',
+    },
+    remove_confirm: {
+        id: t('mobile.manage_members.remove'),
+        defaultMessage: 'Remove',
+    },
+});
+
+type Props = {
+    canRemoveUser: boolean;
+    channelId: string;
+    manageOption: ManageOptionsTypes;
+    testID?: string;
+    userId: string;
+}
+
+const ManageMembersLabel = ({canRemoveUser, channelId, manageOption, testID, userId}: Props) => {
+    const intl = useIntl();
+    const {formatMessage} = intl;
+    const serverUrl = useServerUrl();
+
+    const handleRemoveUser = useCallback(async () => {
+        await removeMemberFromChannel(serverUrl, channelId, userId);
```

Comment:
Does it work fine with no `await`?

Reply:
yes.  and the I was able to remvoe the awaits on the other suggestions also.